src/api.coffee

```diff
@@ -17,9 +17,15 @@ module.exports = (application) ->
 	parsedRouter.use(bodyParser())
 
 	api.use (req, res, next) ->
+		queryKey = req.query.apikey
+		header = req.get('Authorization')
+		match = header.match(/^ApiKey ([a-f\d]{64})$/)
```

Comment:
I don't think it makes sense for this to check for a fixed length/characters, for instance it won't work with the dev api key of 'bananas'